validator/src/commands/run/args/send_transaction_config.rs

```diff
@@ -0,0 +1,324 @@
+use {
+    crate::commands::{Error, FromClapArgMatches, Result},
+    clap::{value_t, ArgMatches},
+    solana_send_transaction_service::send_transaction_service::{
+        Config as SendTransactionServiceConfig, MAX_TRANSACTION_SENDS_PER_SECOND,
+    },
+};
+
+impl FromClapArgMatches for SendTransactionServiceConfig {
+    fn from_clap_arg_match(matches: &ArgMatches) -> Result<Self> {
+        let batch_send_rate_ms = value_t!(matches, "rpc_send_transaction_batch_ms", u64)?;
+        let retry_rate_ms = value_t!(matches, "rpc_send_transaction_retry_ms", u64)?;
+        if batch_send_rate_ms > retry_rate_ms {
+            return Err(Error::Dynamic(Box::<dyn std::error::Error>::from(format!(
+                "the specified rpc-send-batch-ms ({batch_send_rate_ms}) is invalid, it must be <= \
+                 rpc-send-retry-ms ({retry_rate_ms})"
+            ))));
+        }
+
+        let batch_size = value_t!(matches, "rpc_send_transaction_batch_size", usize)?;
+        let millis_per_second = 1000;
+        let tps = batch_size as u64 * millis_per_second / batch_send_rate_ms;
+        if tps > MAX_TRANSACTION_SENDS_PER_SECOND {
+            return Err(Error::Dynamic(Box::<dyn std::error::Error>::from(format!(
+                "either the specified rpc-send-batch-size ({batch_size}) or \
+                 rpc-send-batch-ms ({batch_send_rate_ms}) is invalid, 'rpc-send-batch-size * 1000 \
+                 / rpc-send-batch-ms' must be smaller than ({MAX_TRANSACTION_SENDS_PER_SECOND}) .",
```

Comment:
Looks CI failed and wants you for format this line too. Probably due to running `cargo fmt` before renaming `rpc_send_batch_size` to `batch_size`